### PR TITLE
Vote proxy factory

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/VoteProxy.sol [hunk 1]
```diff
@@ -1,60 +1,47 @@
...
+    DSToken public iou;
+    DSChief public chief;
+
+    constructor(DSChief _chief, address _cold, address _hot) public {
+        chief = _chief;
+        cold = _cold;
+        hot = _hot;
+        
+        gov = chief.GOV();
+        iou = chief.IOU();
```
Comment (jparklev) [inline src/VoteProxy.sol:20]: Just realized that we no longer need to store this in contract storage. Not a major issue though

Comment (gbalabasquer) [reply src/VoteProxy.sol:20]: why did you need before?

Comment (jparklev) [reply src/VoteProxy.sol:20]: push IOUs to polling w/o loading the address from chief

Comment (gbalabasquer) [reply src/VoteProxy.sol:20]: ahh ok just the `iou` as the `gov` is still being used. But I'd keep in both or none of them.

**Pull request merged**